libr/io/p/io_r2k_linux.c

```diff
@@ -344,7 +344,7 @@ int WriteMemory (RIO *io, RIODesc *iodesc, int ioctl_n, size_t pid, ut64 address
 		}
 
 		memcpy (data.buff, buf, len);
-		ret = ioctl (iodesc->fd, ioctl_n, &data);
+		ret = ioctl ((int)iodesc->data, ioctl_n, &data);
```

Comment:
cache this value as long as its used several times and requires double casting thru size_t